### PR TITLE
2020 updates for new major release

### PR DESCRIPTION
**Pull request by illepic**

### CODE REVIEW

**Review by hugonaili — APPROVED** [review]

**Comment by timcosgrove** [comment]
I will test against ex isting sites.

**Comment by illepic** [comment]
@timcosgrove I see this in CI and tackling now:

image

**Comment by illepic** [comment]
Looks like we're good for Node 8 and 10, but since this breaks on Node 6, we'll release this as `p2-theme-core v11.0.0`.

image

File: package.json [hunk 1]
```diff
@@ -26,38 +26,39 @@
   "license": "MIT",
   "dependencies": {
     "autoprefixer": "^6.3.6",
-    "browser-sync": "^2.26.5",
```
Comment (timcosgrove) [inline package.json:29]: Above here in the file: version number needs to be bumped.

**Pull request merged**